packages/plugin-ext/src/main/browser/file-system-main.ts

```diff
@@ -127,3 +151,28 @@ export class FSResource implements Resource {
 
     dispose(): void { }
 }
+
+/** Resource that delegates to theia FileSystem */
+export class FileResource implements Resource {
```

Comment:
there is already ResourceResolver for file scheme. why it cannot be used?

Reply:
I think it's because as we had injected a resolver I used it but we should use as well `ResourceProvider`, will fix but I've another question (will append for the whole PR)